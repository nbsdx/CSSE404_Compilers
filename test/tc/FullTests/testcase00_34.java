// INVALID

class A
{
   public static void main( String[] args )
   {
      boolean b = (new B()).start(this);
   }
}

class B
{
   public boolean start(A a)
   {
      return false;
   }
   
   public C stop()
   {
	  return new C();
   }
}