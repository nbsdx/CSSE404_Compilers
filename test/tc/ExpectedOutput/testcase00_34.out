Cannot find class named C
Variable this is not declared.
Cannot instantiate undeclared class named C
